pkg/virtctl/utils/utils.go

```diff
@@ -2,11 +2,12 @@ package utils
 
 import (
 	"fmt"
+
+	"golang.org/x/term"
```

Comment:
can you put this please as the last group ?
also the fmt should be still in the group it was

Reply:
@oshoval 
okay. Applied!
I think this project needs to be careful about using "make generate"...

Reply:
Thanks
we have `pull-kubevirt-generate`
and without the right updated bazel files, it fails compilation